Assignment2/JavierBrooktec.ttl

```diff
@@ -0,0 +1,10 @@
+@base <http://example.com/> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#>.
+@prefix : <http://example.com/> .
+:Class01  		:includes :Computer101;
+                :includes :Sensor029.
+:Computer101	:hasOwner :User10A.
+:User10A         :hasName "Pedro" ^^xsd:string.
+:Sensor029  	:hasMeasurement :Measurement8401.
+:Measurement8401 :hasTemperature "29" ^^xsd:integer;
```

Comment:
Syntax errors here. Please check what other students have written here, and parse your files always before submitting

Reply:
ok thanks @ocorcho